src/Dfe.PlanTech.Web/ViewBuilders/RecommendationsViewBuilder.cs

```diff
@@ -79,15 +79,17 @@ bool useChecklist
             NextChunk = nextRecommendationChunk,
             CurrentChunkPosition = currentRecommendationIndex + 1,
             TotalChunks = recommendationChunks.Count,
-            SelectedStatusKey = currentRecommendationHistoryStatus?.NewStatus ?? RecommendationConstants.DefaultStatus,
+            SelectedStatusKey = currentRecommendationHistoryStatus?.NewStatus ?? RecommendationStatus.NotStarted.GetDisplayName(),
```

Comment:
Ditto here - we appear to be conflating/bundling the key with display name.

Presumably we should pass the enum/key around wherever possible and only convert to the display name for display purposes?